tasks/post-to-cmr/index.js

```diff
@@ -167,9 +183,124 @@ function buildOutput(results, input, granules, regex) {
     }
   });
 
-  return Object.keys(granulesHash).map((k) => granulesHash[k]);
+  return granulesHash;
+}
+
+/**
+* Move all files in a collection of granules from staging location fo final location
+*
+* @param {Object} granulesObject - an object of the granules where the key is the granuleId
+* @param {string} sourceBucket - source bucket location of files
+* @returns {Promise} promise resolves when all files have been moved
+**/
+async function moveGranuleFiles(granulesObject, sourceBucket) {
+  const moveFileRequests = [];
+
+  Object.keys(granulesObject).forEach((granuleKey) => {
+    const granule = granulesObject[granuleKey];
+    granule.files.forEach((file) => {
+      const source = {
+        Bucket: sourceBucket,
+        Key: `${file.fileStagingDir}/${file.name}`
```

Comment:
Should `file.fileStagingDir` have a default in case developers forget to set it?